apps/courses/src/theme.ts

```diff
@@ -223,6 +223,28 @@ export default extendTheme({
     header: '4.5rem',
   },
   components: {
+    Accordion: {
+      variants: {
+        spaced: {
```

Comment:
Is this the best name here? Let's discuss this - I wonder if "numbered" could be better. I'm not sure - let's talk about what makes this variant unique and then we can figure out how it should be done.

Reply:
Reminder: Create the `NumberedAccordion` component